src/org/zaproxy/zap/extension/ascanrulesBeta/SourceCodeDisclosureSVN.java

```diff
@@ -212,7 +228,23 @@ public int getCweId() {
 	public int getWascId() {
 		return 34;  //Predictable Resource Location
 	}
-
+	
+	private boolean shouldStop(AlertThreshold alertThreshold, Integer statusCode) {
```

Comment:
The type of parameter statusCode could be primitive type int (avoids unnecessary un/boxing conversions).